uturn/app/uturn.hoon

```diff
@@ -0,0 +1,75 @@
+/-  uturn
+/+  default-agent, dbug
+=,  hmac:crypto
+|%
++$  versioned-state
+    $%  state-0
+    ==
++$  state-0  [%0 server-config=(unit server:uturn) threads=(map @ta [=path])]
++$  card  card:agent:gall
+--
+%-  agent:dbug
+=|  state=state-0
+^-  agent:gall
+=<
+|_  =bowl:gall
++*  this     .
+    default   ~(. (default-agent this %|) bowl)
+    helper    ~(. +> bowl)
+::
+++  on-init
+  `this
+++  on-save
+  ^-  vase
+  !>(state)
+++  on-load  :: on-load:default
+  |=  =vase
+  ^-  (quip card _this)
+  =/  old-state  !<(versioned-state vase)
+  ?-  -.old-state
+    %0  `this(state old-state)
+  ==
+++  on-poke
+  |=  [=mark =vase]
+  ^-  (quip card _this)
+  ?+  mark  (on-poke:default mark vase)
+  %set-server-config
+    =/  new-server-config  !<(server:uturn vase)
+    `this(server-config.state (some new-server-config))
+  ==
```

Comment:
```suggestion
  ?+  mark  (on-poke:default mark vase)
      %set-server-config
    =/  new-server-config  !<(server:uturn vase)
    `this(server-config.state (some new-server-config))
  ==
```

Proper [queenside](https://urbit.org/docs/hoon/reference/style#tall-layout-conventions) format. The same change is applicable in `+on-watch`.